.github/workflows/python-dev.yml

```diff
@@ -72,25 +72,5 @@ jobs:
       run: |
         python -c "import pandas; pandas.show_versions();"
 
-    - name: Test with pytest
-      shell: bash -el {0}
-      run: |
-        ci/run_tests.sh
-
-    - name: Publish test results
-      uses: actions/upload-artifact@v3
-      with:
-        name: Test results
-        path: test-data.xml
-      if: failure()
-
-    - name: Report Coverage
-      run: |
-        coverage report -m
-
-    - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v2
-      with:
-        flags: unittests
-        name: codecov-pandas
-        fail_ci_if_error: true
+    - name: Test
```

Comment:
Could you link to the build where this was run (currently this is disabled)?

Reply:
(Not in the yaml, but here in the comments)

Reply:
Can't find it, will re-run. Using NumPy stable because nightly was broken the other day.

Reply:
There seem to be errors that I'm not sure are related to this PR. Could you have a look?

Also I realised that in the dev build we have `fail_ci_if_error: true` (https://github.com/pandas-dev/pandas/blob/44b660dc4a07f4fb507c31795ae63dca2e6e9440/.github/workflows/python-dev.yml#L96) but everywhere else we have `fail_ci_if_error: false`. Shall I make this a flag of the `run-tests` action?

Reply:
Thanks looks like the tests are running with expected failures.

Also yes, `fail_ci_if_error: false` should be set in the `run-tests` action.

Reply:
Set to `false` for all invocations or keep current behaviour?

Reply:
`fail_ci_if_error: false` should be used for all `codecov/codecov-action@v2` actions invocations